src/index.html

```diff
@@ -17,16 +17,24 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form">
+      <div class="form__icon form__icon--big">
+        <input
+          class="input input__big"
+          type="text"
+          data-qa="keypress big"
```

Comment:
```suggestion
    <form class="form" data-qa="big">
      <div class="form__icon form__icon--big">
        <input
          class="input input__big"
          type="text"
          data-qa="keypress"
```

all tests failed.
pay attention on task description and place data-qa attributes according to description
![Screenshot 2024-10-05 at 09 03 56](https://github.com/user-attachments/assets/14e59557-7d9a-45b8-bbb0-b1a04685e56b)
![Screenshot 2024-10-05 at 09 06 23](https://github.com/user-attachments/assets/6a27b89e-13e9-4d9f-b40d-97a7ed61c230)